packages/block-library/src/cover/edit.js

```diff
@@ -309,8 +309,6 @@ function CoverEdit( {
 	overlayColor,
 	setAttributes,
 	setOverlayColor,
-	toggleSelection,
```

Comment:
The `toggleSelection` is available as a block edit component prop; I'm not sure why we were overriding it.

https://github.com/WordPress/gutenberg/blob/11b0d32adc300dae20d00228927a2719cafb53f2/packages/block-editor/src/components/block-list/block.js#L115-L117

Reply:
Good catch! I guess it might not be available at the time this was written? 🤔 Also it's easy to miss some of these properties that are available 😄 I updated.